src/Imath/ImathMatrix.h

```diff
@@ -800,6 +800,8 @@ template <class T> class IMATH_EXPORT_TEMPLATE_TYPE Matrix33
 template <class T> class IMATH_EXPORT_TEMPLATE_TYPE Matrix44
 {
 public:
+    using value_type = T;
```

Comment:
`T` is available as `BaseType`, declared on line #1302, will that suffice? I know `value_type` is more modern and standard, but better to stick with the convention. Same applies in ImathQuat.h. Frustum doesn't have the same declaration, but for consistency it should probably be the same name.

Reply:
Ah I didn't see it, that will do! Thanks!